setup.cfg

```diff
@@ -56,3 +56,10 @@ test =
 [options.entry_points]
 console_scripts =
     voila = voila.app:main
+
+[check-manifest]
```

Comment:
We already have `check-manifest` setting in  `pyproject.toml`

Reply:
Oh I missed it, thanks!